binancial/strategies/mean_reversion_crossover.py

```
@@ -1,28 +1,33 @@
-def mean_reversion_crossover(ticker, accountant, price_usdt, timestamp):
+class MeanReversionCrossover:
+    def __init__(self, sell_threshold=1.01):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)